docs/modules/ROOT/pages/advanced-how-tos/using-trusted-artifacts.adoc

```diff
@@ -0,0 +1,38 @@
+= Using Trusted Artifacts
+
+Konflux relies on Trusted Artifacts to securely allow sharing of data between Tasks without
+compromising the build process. See
+link:https://konflux-ci.dev/architecture/ADR/0036-trusted-artifacts.html[ADR-36] for more
+information.
+
+Today, when a component is onboarded to Konflux, the provided build Pipeline uses Trusted Artifacts.
+However, older components might still be using a version of the build Pipeline which does not use
+Trusted Artifacts. This document explains how to identify if Trusted Artifacts are being used and,
+if not, how to start using them.
+
+== Am I using Trusted Artifacts?
+
+To determine if a certain component is using Trusted Artifacts, inspect its build Pipeline. Look for
+the PipelineRun YAML files under the `.tekton` directory in the git repository of the component. If
+the PipelineRun contains a workspace named `workspace`, it is likely NOT using Trusted Artifacts.
+
+== Migrate to Trusted Artifacts
+
+For components already onboarded to Konflux, the simplest way to start using Trusted Artifacts is
+via the link:https://github.com/zregvart/build-migration[build-migration] tool. Run the
```

Comment:
Can we move this tool to somewhere within the konflux-ci GitHub org?

Reply:
https://issues.redhat.com/browse/EC-792